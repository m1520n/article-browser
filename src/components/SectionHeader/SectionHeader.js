import React from 'react';
import PropTypes from 'prop-types';

import { Wrapper, SectionName } from './SectionHeader.style';

const propTypes = {
  sectionName: PropTypes.string.isRequired,
};

const SectionHeader = ({ sectionName }) => (
  <Wrapper>
    <SectionName>{sectionName}</SectionName>
  </Wrapper>
);

SectionHeader.propTypes = propTypes;

export default SectionHeader;
