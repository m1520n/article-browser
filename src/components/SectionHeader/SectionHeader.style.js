import styled from 'styled-components';

import { fonts, colors } from '../../styles/variables';

export const Wrapper = styled.header``;

export const SectionName = styled.h1`
  width: 100%;
  font-family: ${fonts.arialMT};
  color: ${colors.darkSlate};
  font-size: 3.2rem;
`;
