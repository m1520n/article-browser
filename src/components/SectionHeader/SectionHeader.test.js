import React from 'react';
import renderer from 'react-test-renderer';

import SectionHeader from './SectionHeader';

test('SectionHeader is rendered correctly', () => {
  const component = renderer.create(<SectionHeader sectionName="TEST_SECTION" />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
