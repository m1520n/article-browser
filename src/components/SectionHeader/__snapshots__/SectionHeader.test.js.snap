// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader is rendered correctly 1`] = `
<header
  className="sc-bdVaJa bDWFJH"
>
  <h1
    className="sc-bwzfXH jcLPZV"
  >
    TEST_SECTION
  </h1>
</header>
`;
